docs/source/manual.rst

```diff
@@ -0,0 +1,33 @@
+User Manual for ``autojudge``
+=============================
+
+Creating your account / Logging in
+----------------------------------
+
+On the home page, click ``LOG IN`` (see top right corner).
+Login via your IITH Google account.
+
+Basic Ideas for User
```

Comment:
Could benefit from a better title.

“Key abstractions of autojudge” or sth like that.